### PR TITLE
Add --update-file-options flag to format

### PR DESCRIPTION
**Pull request by peter-edge**
This adds a `--update-file-options` flag to `prototool format` and `prototool all` to update the values of `go_package, java_package` to the correct values based on the package per the style guide, and to delete `java_multiple_files` and `java_outer_classname`.

### CODE REVIEW

File: internal/cmd/flags.go [hunk 1]
```diff
@@ -116,3 +117,7 @@ func (f *flags) bindProtocURL(flagSet *pflag.FlagSet) {
 func (f *flags) bindUncomment(flagSet *pflag.FlagSet) {
 	flagSet.BoolVar(&f.uncomment, "uncomment", false, "Uncomment the example config settings.")
 }
+
+func (f *flags) bindUpdateFileOptions(flagSet *pflag.FlagSet) {
+	flagSet.BoolVar(&f.updateFileOptions, "update-file-options", false, "Update the file options go_package, java_package to match the package per the guidelines of the style guide.")
```
Comment (AlexAPB) [inline internal/cmd/flags.go]: nit: replace comma with `and`

File: internal/format/first_pass_visitor.go [hunk 2]
```diff
@@ -39,10 +40,14 @@ type firstPassVisitor struct {
 	Imports []*proto.Import
 
 	haveHitNonComment bool
+
+	updateFileOptions bool
```
Comment (AlexAPB) [inline internal/format/first_pass_visitor.go]: nit: Maybe `fixFileOptions` or `repairFileOptions`?

File: internal/cmd/flags.go (hunk 1 above)
Comment (peter-edge) [reply internal/cmd/flags.go]: Done

File: internal/format/first_pass_visitor.go (hunk 2 above)
Comment (peter-edge) [reply internal/format/first_pass_visitor.go]: I want this to match the flag

**Pull request merged**